lib/fastlane/plugin/wpmreleasetoolkit/helper/github_helper.rb

```diff
@@ -71,6 +72,32 @@ def self.create_release(repository, version, release_notes, assets, prerelease)
           github_client().upload_asset(release[:url], file_path, content_type: 'application/octet-stream')
         end
       end
+
+      # Downloads a file from the given GitHub release
+      #
+      # @param [String] repository The repository name (including the organization)
+      # @param [String] relase The title of the GitHub release
+      # @param [String] file_path The path, inside the project folder, of the file to download
+      # @param [Striog] download_folder The folder which the file should be downloaded into
+      # @return [String] The path of the downloaded file, or nil if something went wrong
+      #
+      def self.download_file_from_release(repository:, release:, file_path:, download_folder:)
```

Comment:
```suggestion
      def self.download_file_from_release(repository:, commit:, file_path:, download_folder:)
```
(see suggestion above; obv also update the impl and call sites)